common/app/com/commercetools/sunrise/framework/viewmodels/forms/QueryStringUtils.java

```diff
@@ -50,6 +48,20 @@ private QueryStringUtils() {
     }
 
     /**
+     * Extracts the query string from the request.
+     * @param httpRequest current HTTP request
+     * @param ignoredParams params that should not be included in the link
+     * @return query string from the request
+     */
+    public static Map<String, List<String>> extractQueryString(final Http.Request httpRequest,final Set<String> ignoredParams) {
+
+        Set<String> ignored = Optional.ofNullable(ignoredParams).orElseGet(HashSet::new);
```

Comment:
In Sunrise we follow the rule that unless a parameter is annotated with `@Nullable`, it can't be null. Therefore this line of code is unnecessary.